pkg/cmd/install_test.go

```diff
@@ -40,8 +63,38 @@ func Test_loadPublickeyEncrypted(t *testing.T) {
 	}
 
 	tmpfile.Close()
-	_, err = loadPublickey(tmpfile.Name())
+	_, _, err = loadPublickey(tmpfile.Name())
 	if err.Error() != expected {
 		t.Errorf("Unexpected error, got: %q, want: %q.", err.Error(), expected)
 	}
 }
+
+func Test_RewriteKubeconfig(t *testing.T) {
+	var ip = "192.168.0.25"
+	var context = "context-test"
+
+	kubeconfig = rewriteKubeconfig(kubeconfigExample, ip, context)
+
+	re := regexp.MustCompile(`server:\s?https://(.*):\d+`)
+	group := re.FindSubmatch(kubeconfig)
+
+	if len(group) == 0 || string(group[1]) != ip {
+		t.Errorf("Unexpected error, got: %q, want: %q.", string(group[1]), ip)
+	}
+
+	re = regexp.MustCompile(`context-test`)
+	match := re.FindAllIndex(kubeconfig, -1)
+
+	if len(match) != 6 {
```

Comment:
I do not like magic numbers like "6" randomly in the code, could you instead break out what you want to match and then check against the length of that?

```
want :="whatever"

if len(match) != len(want) {

}
```